mkosi/__init__.py

```diff
@@ -4054,12 +4054,17 @@ def install_unified_kernel(
                 image_id = f"mkosi-{args.distribution}"
                 partlabel = None
 
+            # See https://systemd.io/AUTOMATIC_BOOT_ASSESSMENT/
+            boot_count = ""
+            if root.joinpath("etc/kernel/tries").exists():
+                boot_count=f'+{root.joinpath("etc/kernel/tries").read_text().strip()}'
```

Comment:
Whitespace is missing around "=".

Please write this as:
```python
boot_count = f'…' if root.joinpath("etc/kernel/tries").exists() else ''
```

Reply:
Done